src/pseudopeople/configuration/generator.py

```diff
@@ -132,6 +132,13 @@ def _generate_configuration(is_no_noise: bool) -> ConfigTree:
                 else:
                     noise_level = row_noise.row_probability
                 row_noise_type_dict[Keys.ROW_PROBABILITY] = noise_level
+            if row_noise.additional_parameters is not None:
+                for key, value in row_noise.additional_parameters.items():
+                    if is_no_noise:
```

Comment:
Is this what we actually want to do? I don't remember what the `is_no_noise` parameter refers to, but in the column noise generator we don't do this for the `additional_parameters`. What you've got here would work for the guardian-based duplication noise function, but it would break down if we ever added a row noise function that could take an additional parameter that isn't a float value.

If there isn't an easy way to navigate that, I think it's sufficient to put a `FIXME` calling out this concern.

Reply:
is_no_noise is a bool that determines if we set the noise level to 0. All column noise types have a cell probability which gets set to 0 when is_no_noise is True which is why we do not need to do it there. You are correct about this short coming and I will call it out but we currently do not have other row noise types with additional parameters so it is hard to anticipate how to handle any future implementations at the moment.